### ISSUE TITLE
Reintroduce Mood Images to PLPs

### ISSUE THREAD

**Issue by ErinLOD** [Land-of-Distraction/LoD.com#19]
once our source set & load time are solid

**Comment by ErinLOD** [comment]
@briznad do you think we're in a solid load time place yet for me to start reintroducing mood images into the PLPs?

**Comment by briznad** [comment]
I'm going to capture some thoughts exchanged via email regarding this issue:

> Re: mood images; yes, I think we can add these back in if that's desired. I don't think they necessarily pose a risk to page load times. I believe they were originally removed because of a reported flickering problem. It's possible that adding them back will reintroduce this issue, but I won't know until I try. As far as how we reintroduce the mood images, the way the mood images are loaded is they are created as a "dummy" product with the tag "in grid promo". If you search for this tag on the products page you can see that all of the mood image products have been deleted, save for 1 pertaining to leather. Should I copy over the mood image products from the testing site?

**Comment by briznad** [comment]
@ErinLOD Should I copy over the mood images from the testing site?

**Comment by ErinLOD** [comment]
No, I think we need new ones. Those we're some samples selected by the development agency.


Thanks!


Erin Frederick

erin@landofdistraction.com

858.531.2112


This email is  confidential. If you are not the intended recipient, any use, copying, disclosure, or distribution is  prohibited.  Please notify us immediately and delete this email from your computer systems. Unless this email is sent pursuant to an agreement between Distraction Wear Inc. or any of its affiliates, and the recipient, Distraction Wear Inc. and its affiliates do not provide any warranties, representations and conditions with respect to this email and its contents, and disclaim any such warranties, representations and conditions, express or implied. Distraction Wear Inc. and its affiliates do not assume any liability with respect to the accuracy or completeness of the information set out in this email, or for any use of, or inability to use any of the information set out in this email.  If you do not wish to receive any further emails from us, you may unsubscribe by replying “unsubscribe”.  You may also contact us in writing at the following address: stacey@landofdistraction.com<mailto:stacey@landofdistraction.com>

________________________________

**Comment by briznad** [comment]
@ErinLOD let me know when you've located the new assets to be used. 

also, if you check out the previous abstract images on the testing site (https://land-of-distraction.myshopify.com/admin/products?query=in+grid+promo) you can see that different images were associated with/shown alongside various product types by setting the image to that product type (no product type seems to mean show with everything). so when you know which images you'd like me to use can you also supply a list of which images should be associated with which products?

**Comment by ErinLOD** [comment]
Hi Brad,


So i originally added all the mood images to the PDPs on the dev site as products tagged 'in grid promo' and the collection that we want them to be displayed in so I think it might be less work for me to just do it that way again instead of supplying them to you with the criteria.


Thanks!


Erin Frederick

erin@landofdistraction.com

858.531.2112


This email is  confidential. If you are not the intended recipient, any use, copying, disclosure, or distribution is  prohibited.  Please notify us immediately and delete this email from your computer systems. Unless this email is sent pursuant to an agreement between Distraction Wear Inc. or any of its affiliates, and the recipient, Distraction Wear Inc. and its affiliates do not provide any warranties, representations and conditions with respect to this email and its contents, and disclaim any such warranties, representations and conditions, express or implied. Distraction Wear Inc. and its affiliates do not assume any liability with respect to the accuracy or completeness of the information set out in this email, or for any use of, or inability to use any of the information set out in this email.  If you do not wish to receive any further emails from us, you may unsubscribe by replying “unsubscribe”.  You may also contact us in writing at the following address: stacey@landofdistraction.com<mailto:stacey@landofdistraction.com>

________________________________

**Comment by briznad** [comment]
@ErinLOD sounds good
